examples/interpolation_surrogate/4d_interp.C

```diff
@@ -104,6 +105,12 @@ int main(int argc, char ** argv)
   // desired points in parameter space.
   builder.build_values();
 
+  // Now that we've built the data, we write it out so we can reuse it later
+  QUESO::InterpolationSurrogateIOASCII<QUESO::GslVector, QUESO::GslMatrix>
+    data_writer;
+
+  data_writer.write( "4d_interp_data.dat", data );
```

Comment:
What happens if you try to write/read something that hasn't yet been built by the builder?